web/src/features/panels/zone/ZoneHeader.tsx

```diff
@@ -1,6 +1,6 @@
 import CarbonIntensitySquare from 'components/CarbonIntensitySquare';
 import { CircularGauge } from 'components/CircularGauge';
-import ZoneHeaderTitle from './ZoneHeaderTitle';
+import ZoneHeaderTitle from 'features/panels/zone/ZoneHeaderTitle';
 
```

Comment:
Did you editor change this? 🤔  It should be the relative path when inside same folder
```suggestion
import ZoneHeaderTitle from './ZoneHeaderTitle';
```

Reply:
Fixed